packages/ignite/src/commands/new.js

```diff
@@ -34,6 +34,25 @@ const installWalkthrough = [
   }
 ]
 
+const minOptions = {
+  'dev-screens': 'No',
+  'vector-icons': 'none',
+  'i18n': 'none',
+  'animatable': 'none'
+}
+const maxOptions = {
+  'dev-screens': 'Yes',
+  'vector-icons': 'react-native-vector-icons',
+  'i18n': 'react-native-i18n',
+  'animatable': 'react-native-animatable'
+}
```

Comment:
@kevinvangelder had pointed this out as a good candidate for refactoring.  Thanks for cleaning this up.  Love this.